week-1/exercises/initials.js

```diff
@@ -4,3 +4,5 @@ let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first character of each string in upper case to form the user's initials
 // Log the variable in each case
+initials = firstName.charAt(0) + middleName.charAt(0) + lastName.charAt(0);
```

Comment:
The comment on line 5 asks to store the first character of each string in upper case. What would happen if I set my `firstName` to be "creola"?

Reply:
You broke my code, I fixed it, I used the function toUpperCase(), and before I saved it in another variable the initials letter for each name.